src/Storage/Storage.Management/StorageAccount/NewAzureStorageAccount.cs

```diff
@@ -22,9 +22,12 @@
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using System;
 using System.Collections.Generic;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Management.Storage
 {
+    [GenericBreakingChange("Default behavior of AllowBlobPublicAccess will be changed", OldWay = "AllowBlobPublicAccess is set to True by defult.", 
```

Comment:
"Default behavior of AllowBlobPublicAccess will be changed" -> "Default behavior of AllowBlobPublicAccess will be changed in a future release."

And it's better to add the impact of the default value change, like after the change, with default value as false, user can't set the container public access, or the public access setting on container won't work (not allow anonymous access to containers and blobs)? Please test which way is correct and update it.

Reply:
Done. Can you help review if the updated warning message detailed enough? 

![image](https://user-images.githubusercontent.com/100746763/212841703-99df0689-da94-434e-9c16-20c1c199f639.png)